src/MediaAtom/Caption.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+
+import { text } from '@guardian/src-foundations/palette';
+import { from, until } from '@guardian/src-foundations/mq';
+import { textSans } from '@guardian/src-foundations/typography';
+import { space } from '@guardian/src-foundations';
+import { css, cx } from 'emotion';
+import { pillarPalette } from './pillarPalette';
+import { Format, Pillar, Display } from '@guardian/types/Format';
+
+const TriangleIcon = (
+    <svg width="11" height="10" viewBox="0 0 11 10">
+        <path fillRule="evenodd" d="M5.5 0L11 10H0z" />
+    </svg>
+);
+
+type Props = {
+    format: Format;
```

Comment:
Feels weird to have to bring caption into here when it exists in dotcom rendering? I would say that the caption used for all embeds/images/media etc should be the same? +1 on taking a Format!